include/deal.II/grid/tria.h

```diff
@@ -28,13 +28,6 @@
 // Ignore deprecation warnings for auto_ptr.
 DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
 #include <boost/signals2.hpp>
-#include <boost/serialization/vector.hpp>
-#include <boost/serialization/map.hpp>
-#ifdef DEAL_II_WITH_CXX11
-#  include <boost/serialization/unique_ptr.hpp>
```

Comment:
Yes, you are right. I was under the impression that compiling and searching for `boost::vector` and `boost::map` would be sufficient to be convinced that these headers were not used. The testsuite proved me wrong...